### ISSUE TITLE
unable to boot after executing  phh-securize.sh

### ISSUE THREAD

**Issue by TheFrozenDuck** [phhusson/treble_experimentations#2370]
### Current Behavior:
  
When trying to remove the default superuser by executing `phh-securize.sh` the device gets stuck after rebooting. 

### Describe the bug
 
After execution of the script `phh-securize.sh` the device reboots. After it reboots the device gets stuck at the Samsung logo. No matter if the data is wiped or a factory reset is done using TWRP, the device will not boot.

### To Reproduce
 
1. Flash a GSI ROM
2. execute `adb shell`
3. in the spawned shell execute `sh /system/bin/phh-securize.sh`
4. wait for the secript to finished and reboot the device
5. the device boots and gets stuck at the logo
6. (optional) reboot into recovery and wipe data/factory reset
7. (optional) reboot and the device boots and gets stuck at the logo

### Expected behavior
 
The device continues to boot after the script has been executed

### Photos
 
n/a

### Smartphone (please complete the following information):
 - Device: Samsung S20 FE 5G 
 - Build Flavor: system-squeak-arm64-ab-vndklite-vanilla (not entirely sure, a ROM building on it (LineageOS) is used)
 - Version: most likely 414 (not entirely sure, a ROM building on it (LineageOS) is used)

### Additional context
 
n/a

**Comment by phhusson** [comment]
you probably need to do a factory reset after securize - which is why
nowadays it's recommended to use secure variant directly

Le dim. 19 juin 2022 à 12:33, TheFrozenDuck ***@***.***> a
écrit :

**Comment by TheFrozenDuck** [comment]
Thanks for your fast answer. I did, as desecribed, a factory reset together with wiping data and it still lead to the problem.

If I am not mistaken some years ago one was advised to boot directly into recovery and executing a factory reset. Do you think that may be required in this case as well?

**Comment by phhusson** [comment]
... How did you do the factory reset I told you to do if not via recovery?!?

19 juin 2022 12:53:17 TheFrozenDuck ***@***.***>:

**Comment by TheFrozenDuck** [comment]
I did it in the recovery. My bad for the bad description:

With the " boot directly into recovery" I meant that the script is executed and device rebooted directly into the recovery, rather than let it boot into the system, get stuck and the rebooting into recovery.

I was able to narrow the problem down to the remount but at the same time as I looked into that I happened to stumble on your sas-creator. The resulting image worked for me.
